modules/deploy/pages/deployment-option/cloud/vpc-peering-aws.adoc

```diff
@@ -23,15 +23,15 @@ The following command routes traffic from Redpanda to AWS by finding the route t
 
 [,bash]
 ----
-aws ec2 describe-route-tables --filter "Name=tag:Name,Values=network-<redpanda-cluster-id>" "Name=tag:purpose,Values=private" | jq -r '.RouteTables[].RouteTableId' | \
+aws ec2 describe-route-tables --filter "Name=tag:Name,Values=network-<redpanda-network-id>" "Name=tag:purpose,Values=private" | jq -r '.RouteTables[].RouteTableId' | \
 while read -r route_table_id; do \
 aws ec2 create-route --route-table-id $route_table_id --destination-cidr-block <aws-vpc-cidr-block> --vpc-peering-connection-id <peering-connection-id>; \
 done;
 ----
 
 Replace the following placeholder values:
 
-* Redpanda cluster ID: This is listed in the *Overview* page of your cluster.
+* Redpanda network ID: This is listed in the Redpanda Cloud UI after clicking on the name of the *Redpanda network* in the *Details* section of the *Overview* page of your cluster.
```

Comment:
I'd go as far as to say we should explicitly say "Note: although this may be similar, this is distinct from your Cluster's ID" just to put people on alert that these two are very similar.

Reply:
Note: This network ID may look similar, however, it is distinct from your cluster ID.

Reply:
Thanks for the suggestions, I have just updated the doc